kde-plasma/plasma-desktop/plasma-desktop-5.27.7.1-r1.ebuild

```diff
@@ -50,7 +50,7 @@ COMMON_DEPEND="
 	>=kde-frameworks/kdbusaddons-${KFMIN}:5
 	>=kde-frameworks/kdeclarative-${KFMIN}:5
 	>=kde-frameworks/kded-${KFMIN}:5
-	>=kde-frameworks/kdelibs4support-${KFMIN}:5
+	>=kde-frameworks/kdelibs4support-${KFMIN}:5[handbook?]
```

Comment:
This is likely fighting a symptom with an incorrect USEdep. plasma-desktop would only need this if it were using a macro from kdelibs4support. Does it? How exactly did it fail or what are you trying to fix here?

Reply:
I can build plasma-desktop with `-handbook` perfectly fine right now, so there is nothing to fix unless I would break kdelibs4support like you did ;)